Source/Dafny/Compiler-js.cs

```diff
@@ -21,6 +21,12 @@ public JavaScriptCompiler(ErrorReporter reporter)
 
     public override string TargetLanguage => "JavaScript";
 
+    protected readonly new string DafnySetClass = "_dafny.Set";
+    protected readonly new string DafnyMultiSetClass = "_dafny.MultiSet";
+    protected readonly new string DafnySeqClass = "_dafny.ISequence";
+    protected readonly new string DafnySeqHelperClass = "_dafny.Sequence";
+    protected readonly new string DafnyMapClass = "_dafny.Map";
```

Comment:
Evidently, this is how the Java compiler did it. I recently saw that the new C++ compiler copied this style, and so does this JavaScript change. I think these fields should all have been declared virtual in Compiler.cs, along the lines of:

    protected virtual string DafnySetClass => "Dafny.Set";

Then, compilers that want to use different names would do:

    protected override string DafnySetClass => "dafny.DafnySet";

Reply:
Fixed